setup.py

```diff
@@ -28,9 +28,9 @@ def find_version(*file_paths):
 
 
 setup(
-    name="netmiko",
+    name="netmiko-bti-4-1-1",
```

Comment:
I assume this accidental/not intended. Removing it.

Reply:
Yes sorry, I used the code to create a package for local installation and publish to an internal pypy, and I updated the name and description. Sorry I should have reverted it back